### PR TITLE
LPS-141380 LPS-139191 Translation manager bugs

### PR DESCRIPTION
**Pull request by tinycarol**
### References

- [LPS-141380](https://issues.liferay.com/browse/LPS-141380)
- [LPS-139191](https://issues.liferay.com/browse/LPS-139191)

### What is the goal?

* Fix add languages button focus outline being partially hidden
* Fix delete languages buttons not being keyboard-accessible

#### Before PR

https://user-images.githubusercontent.com/6642927/139438678-faad1dcb-1ad5-489a-8b73-b786ac4202bb.mov

https://user-images.githubusercontent.com/6642927/139438693-dd605531-f06f-41c1-b132-957ba9363cab.mov

#### After PR

https://user-images.githubusercontent.com/6642927/139438734-08add1f3-59c4-4561-aedd-38ed559f78ca.mov

https://user-images.githubusercontent.com/6642927/139438737-5110e807-b392-457d-83b4-5f2a1d9d7a4d.mov

### How to replicate
* Go to Content & Data > Web Content
* Click on translations button
* Click on `Manage Translations`
* Click on `+` button, see that the focus outline is not cropped
* Click outside the dropdown so it will close
* Use the tab keyboard key to move around the modal, see that the trash buttons are now focusable

### Checklist

- ~~[ ] I have made corresponding changes to the documentation~~
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked this works in the corresponding browser

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js [hunk 1]
```diff
@@ -71,72 +71,66 @@ const TranslationAdminContent = ({
...
+				<ClayInput.Group className="align-items-center">
+					<ClayInput.GroupItem>
+						<ClayInput
+							aria-label={Liferay.Language.get('search')}
+							insetAfter={true}
+							onChange={(event) => {
+								const {value} = event.target;
+
+								setSearchValue(value);
+							}}
```
Comment (diegonvs) [inline modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js:83]: If you want to decrease the quantity of lines here, you can:

```suggestion
							onChange={({target: value}) => 
								setSearchValue(value)
							}
```

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js [hunk 2]
```diff
@@ -203,15 +197,19 @@ const TranslationAdminContent = ({
 
 									<ClayTable.Cell>
 										{!isDefaultLocale && (
-											<ClayIcon
-												className="inline-item"
+											<ClayButton
```
[the remark this answers is not in the document]
Comment (victorg1991) [reply modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js:200]: It seems that ClayButtonWithIcon set monospaced property to true by default: https://github.com/liferay/clay/blob/master/packages/clay-button/src/ButtonWithIcon.tsx#L24, so you can use ClayButtonWithIcon setting that explicitly to false:

```jsx
<ClayButtonWithIcon
    monospaced={false}
    symbol="trash"
/>
```

**Review by victorg1991 — APPROVED** [review]
LGTM! :)

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js (hunk 1 above)
Comment (tinycarol) [reply modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminContent.js]: Done! Thanks ^^

**Comment by carloslancha** [comment]
LNHTM!

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**